app/gui/src/dashboard/components/AriaComponents/Dialog/Dismiss.tsx

```diff
@@ -0,0 +1,41 @@
+/** @file A button to close a dialog without submitting it. */
+import type { JSX } from 'react'
+
+import { Button, useDialogContext, type ButtonProps } from '#/components/AriaComponents'
+import { useText } from '#/providers/TextProvider'
+
+/** Additional props for the Cancel component. */
+interface DialogDismissBaseProps {
+  readonly variant?: ButtonProps['variant']
+}
+
+/** Props for a {@link DialogDismiss}. */
+export type DialogDismissProps = DialogDismissBaseProps &
+  Omit<ButtonProps, 'formnovalidate' | 'href' | 'variant'>
+
+/** Dismiss button for dialogs. */
+export function DialogDismiss(props: DialogDismissProps): JSX.Element {
+  const { getText } = useText()
+
+  const { size = 'medium', ...buttonProps } = props
+
+  const dialogContext = useDialogContext()
+
+  return (
+    <Button
+      testId="form-cancel-button"
+      formnovalidate
+      type="button"
+      variant="outline"
+      size={size}
+      onPress={() => {
```

Comment:
We need to stack onPress with the one that comes from the props

Reply:
good catch, thanks, fixed